articles/laravel-8-email-verification/index.md

```diff
@@ -46,24 +46,28 @@ cd verifyEmailApp
 php artisan serve
 ```
 
+`php artisan serve` command allows the application to be hosted locally.
+
 **Using Laravel Installer**  
-To build Laravel applications easily, Laravel provides a helpful command-line utility. Download the installer first: 
+There are numerous command-line utilities that make it easy to build applications using Laravel. We use the following command to download the Laravel installer: 
 
 ```bash
 composer global require laravel/installer
 ```
 
 To run the Laravel installer, you need to make sure that the Composer binary folder is within your `$PATH` variable.
-See if it is already in your `$PATH` variable, first by running the following command in your terminal:  
+Check if it is in your `$PATH` variable by running the following command in your terminal:  
 
 ```bash
 echo $PATH
 ```
-The performance should contain something like this if all is correct:  
+
+The correct output should look, as shown below:  
 `User/username/.vendor/composer/bin`  
-If not, change your `.bashrc` or, if you're using `ZSH`, your `.zshrc` so that it includes the path to the vendor directory for your composer.  
+
+Incase of any errors, change your `.bashrc` or, if you're using `ZSH`, your `.zshrc` so that it includes the path to the vendor directory for your composer.  
 When installed, this command creates a new installation for Laravel in the directory you choose.  
-It's even possible to use. (a dot) instead of `[foldername]` to construct a project without having a subdirectory in the current working directory.
+It's even possible to use a fullstop instead of `[foldername]` to construct a project without having a subdirectory in the current working directory.
```

Comment:
While dealing with directories, we can't talk about 'fullstop' but a 'dot' instead!

Reply:
A beginner cannot know what a dot is. Full stop makes better sense.

Reply:
Not convinced! Folder structures cannot be referred to by a mere 'full-stop.

Reply:
```
It's even possible to use. (a dot) instead of `[foldername]` to construct a project without having a
```
What is that `dot` or this `.` called?